backend/codesirius/problems/admin.py

```
@@ -1,8 +1,57 @@
+        super().save_model(request, obj, form, change)
+
+
+admin.site.register(Language, LanguageAdmin)
 admin.site.register(Tag)
-admin.site.register(Problem)
+
+
+class ProblemForm(forms.ModelForm):
+    class Meta:
+        model = Problem
+        fields = "__all__"
```

Ruff (0.8.2)

Do not use `__all__` with `ModelForm`, use `fields` instead

(DJ007)